release/build_release.sh

```diff
@@ -198,7 +270,9 @@ function print_help {
 	echo "The script reads the following environment variables:"
 	echo ""
 	echo "    LIFERAY_RELEASE_GCS_TOKEN (optional): *.json file containing the token to authenticate with Google Cloud Storage"
+	echo "    LIFERAY_RELEASE_GIT_PREV_REF (required if LIFERAY_RELEASE_SOFT is equals to \"true\"): Previous branch to build from"
 	echo "    LIFERAY_RELEASE_GIT_REF: Git SHA to build from"
+	echo "    LIFERAY_RELEASE_GIT_TOKEN: Token to authenticate with Github"
```

Comment:
@kiwm, should the variable name be `LIFERAY_RELEASE_GITHUB_TOKEN`?

Reply:
I put GIT to match the others variables. For constancy.